pkg/authorizations/authorizations.go

```diff
@@ -0,0 +1,52 @@
+package authorizations
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authoritytypes "github.com/zeta-chain/zetacore/x/authority/types"
+	crosschainTypes "github.com/zeta-chain/zetacore/x/crosschain/types"
+	fungibletypes "github.com/zeta-chain/zetacore/x/fungible/types"
+	lightclienttypes "github.com/zeta-chain/zetacore/x/lightclient/types"
+	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+func AuthorizationTable() map[string]authoritytypes.PolicyType {
```

Comment:
I realize the issue https://github.com/zeta-chain/node/issues/1236 is about having it hardcoded but in the future we could think about having this structure on-chain to update it dynamically.

While I think we don't need to implement it for this PR, I think we should still consider changing the map into a slice so we can persist it in the store in the future

Reply:
9485063a5dcc6ff1f35d38cf0c3f70a92507a9a5